cli.js

```diff
@@ -170,11 +166,73 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   .command('list [options]', 'List all deployed functions', (yargs0) => {
     yargs0
       .usage('Usage: $0 list [options]')
-      .option('json', { describe: 'Output as JSON' })
+      .option('json', {
+        describe: 'Output as JSON',
+        type: 'boolean',
+      })
       .strict();
   }, async (argv) => {
     await handleCommand(argv, listHandler);
   })
+  .command('trigger', 'Manages triggers', yargs0 => yargs0
+      .demandCommand()
+      .command('upsert', 'Creates a new trigger or updates an existing one', (yargs1) => {
+        yargs1
+          .usage('Usage: $0 trigger upsert <trigger name> [options]')
+          .positional('trigger name', {
+            describe: 'Trigger name',
+            type: 'string',
+          })
+          .option('source-uri', {
+            demandOption: true,
+            describe: 'Redis-stream uri',
+            type: 'string',
+          })
+          .option('target-func', {
+            demandOption: true,
+            describe: 'Function to run for each item in the stream',
+            type: 'string',
+          })
+          .option('error', {
```

Comment:
Not a good name, should be `error-handler`.

Reply:
changed